patches/base-path.diff

```diff
@@ -65,41 +52,11 @@ Index: sagemaker-code-editor/vscode/src/vs/code/browser/workbench/workbench.ts
  	}
  
  	private startListening(): void {
- @@ -550,17 +551,6 @@ class WorkspaceProvider implements IWork
```

Comment:
can you double check that reemoving this entire patch is correct

Reply:
yes the function is not get deleted in vscode , ref: https://github.com/microsoft/vscode/blob/release/1.93/src/vs/code/browser/workbench/workbench.ts